pkg/framework/backoff.go

```diff
@@ -16,7 +16,7 @@ const (
 	ShortMaxInterval = 5 * time.Second
 )
 
-func newExponentialBackoff(maxWait, maxInterval time.Duration) *backoff.ExponentialBackOff {
+func NewExponentialBackoff(maxWait, maxInterval time.Duration) *backoff.ExponentialBackOff {
```

Comment:
We can decide it will be other type of backoff so maybe it makes sense to rename it to more generic form like `NewBackOff`.

Also nitpick, capital `O` in BackOff.